src/front-end/src/components/MentorAccepted.js

```diff
@@ -35,7 +35,15 @@ class MentorAccepted extends Component {
       <>
         <Form>
           <h1>
-            <center>These are your accepted mentees:</center>
+            <center>Accepted Mentees:</center>
+            <p4>
+              <center>
+                <i>
+                  Below you can interact and view details about each accepted
+                  mentee
```

Comment:
Maybe remove the word 'interact' since interaction between mentees and mentors happens via email, and the word might confuse the user (possibly change to something like: 'View details about the mentees that you've accepted below')

Reply:
Done!